20250321.md

```diff
@@ -0,0 +1,52 @@
+### Meeting Minutes
+
+**Cardano Atlantic Council Meeting**
+
+**Date:** March 21, 2025 19:00 UTC
+
+**Attendees:** 
+- Beatrice Anihiri
+- Thiago Nunes
+- Adam Dean
+- Mike Hornan
+- Jenny Brito
+- Richard McCracken
+
+**Purpose:** 
+1. The meeting aimed to discuss and vote on a governance proposal concerning the Cardano treasury's net change limit. Adam Dean confirmed that five out of six members were present, meeting the quorum requirement.  They also noted the proposal's impact on the budget.
+
+**Summary:**
+>The Cardano NCL proposal meeting, attended by all CAC members, debated a proposal altering the net change limit, focusing on its constitutionality and ambiguity regarding definitions and timeframes.  Key disagreements centered on the interpretation of "net change limit," "period of time," and the use of "shall" in the proposal, leading to a tentative vote failing to reach consensus threshol to deem it constitutional.  The group decided to postpone the final vote until Monday, after asynchronous deliberation, with a final on-chain vote by April 4th.
+
+**Meeting Points:**
+1. **Resolve Script Hash and Mutability**
+ - Suggested linking the proposal text to an IPFS link to ensure reference stability in case the original text changes.
+- Agreed to post the hashes of the files voted upon in the rationale.
```

Comment:
```suggestion
1. **Resolve Script Hash and Mutability**
  - Suggested linking the proposal text to an IPFS link to ensure reference stability in case the original text changes.
  - Agreed to post the hashes of the files voted upon in the rationale.
```